Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -226,6 +229,38 @@
     "PBF Rule": "pbf"
 }
 
+XPATH_ACTIONS_TYPES_MAP = {
```

Comment:
is this a map for other usecases as well? if not lets add this endpoint to the name of the const